src/fastapi_cloud_cli/utils/api.py

```diff
@@ -13,6 +13,7 @@ def __init__(self) -> None:
 
         super().__init__(
             base_url=settings.base_api_url,
+            timeout=httpx.Timeout(20),
```

Comment:
Took the opportunity to update this too, mainly for staging, but default timeout was 5 seconds 😊